### DISCUSSION TITLE
Needed Adaptors?

### DISCUSSION THREAD

**Discussion by DrCord** [OpenFn/adaptors#735] [General]
It's very likely I will be building an OpenFn adapator as part of a work initiative to build expert level familiarity with OpenFn Lightning. Is there a list of needed adaptors anywhere?

**Comment by josephjclark** [comment]
Now there's an offer we can't refuse!

Thanks for asking. We have a list (well actually a few lists), give us a minute and we'll make a suggestion or two for you.

Just thinking aloud - does it have to be a new adaptor or are you up for modernising/updating something older?

**Comment by christad92** [comment]
Hi @DrCord, 

It's Ayodele here - PM at OpenFN. Thanks for your interest and offer. 

Following up on @josephjclark's comment above, we thought it might be better to take advantage of your availability and offer to improve on our Kobotoolbox Adaptor. Please see the issues below: 

- https://github.com/OpenFn/adaptors/issues/695
- https://github.com/OpenFn/adaptors/issues/603

#695 is considered to be a good first issue that you can pick up. I also wanted to mention that you are welcome to be opinionated about the fixes for #603. @josephjclark will provide guidance where necessary so please feel free to reach out to him and raise a PR for him to review when you have one.

**Reply by DrCord** [reply]
I'm thinking of picking this up on my own time, OpenFn seems like a worthwhile project and I'd like to contribute.

**Reply by josephjclark** [reply]
Thank you, Cord! We certainly like to think so.

The kobo adaptor might seem like a hard entry point, unless you happen to know Kobo Toolbox. On the other hand,  it's an adaptor that _actually_ gets used in the wild, and we have some projects with it coming up. It'll also give you a sense of where we've been and where we're going.

So spending your time on it really would be worthwhile, if you want to.

Having a Gmail or slack integration would be cool for sure, and a fun entry point, but I can't guarantee that it would go on and be used.

I know where I'd prefer to spend my time :)
